docs/application-tutorial/game_of_life.py

```diff
@@ -0,0 +1,511 @@
+import sys
+import time
+import math
+import numpy as np
+
+# Hide pygame prompt
+from os import environ
+
+environ["PYGAME_HIDE_SUPPORT_PROMPT"] = "1"
+
+import pygame
+import argparse
+
+from concrete import fhe
+
+# Constants of the game of life
+weights_method_3b_a = np.array([[1, 1, 1], [1, 0, 1], [1, 1, 0]])
+weights_method_3b_b = np.array([[0, 0, 0], [0, 0, 0], [0, 0, 1]])
```

Comment:
Maybe these can be formatted in a matrix shape for better readability?

(note: you can put a trailing comma to the last element to let black accept it)
```python
weights_method_3b_a = np.array([
    [1, 1, 1],
    [1, 0, 1],
    [1, 1, 0],
])         # ^ this
```

Reply:
Same with other 2d arrays below.

Reply:
I am using black and it's automatically formatted like that :) You're not using black as well, on the CP side?

Reply:
Please tell me if I can close

Reply:
If you add the trailing comma like I said, it'll keep the multi line formatting :slightly_smiling_face:

Reply:
let me try!

Reply:
@umut-sahin , it looks my black (macOS) allows me to do

weights_method_3b_a = np.array(
    [
        [1, 1, 1],
        [1, 0, 1],
        [1, 1, 0],
    ]
)

is it ok with you?

Reply:
I did this latter in the last commit, please tell me if it is fine with you

Reply:
(and if you originally put your "(note: you can put a trailing comma to the last element to let black accept it)", sorry, I didn't see it the first time)

Reply:
No worries, it looks great! Thanks.